specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/azureFirewall.json

```diff
@@ -592,6 +592,13 @@
             "type": "string"
           }
         },
+        "sourceIpTags": {
```

Comment:
Adding this requires incrementing the api-version for this resourceType (same for the other new properties introduced in this file) .  See https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Reply:
Where should I change the api-version of the resourceType?